packages/components/src/internal/components/buttons/DisableableButton.tsx

```diff
@@ -10,31 +14,35 @@ interface Props {
 }
 
 export const DisableableButton: FC<Props> = memo(props => {
-    const { bsStyle = 'default', className = '', disabledMsg, title, onClick, children } = props;
+    const { bsStyle = 'default', children, className = '', disabledMsg, onClick, title } = props;
+    const { onMouseEnter, onMouseLeave, portalEl, show, targetRef } = useOverlayTriggerState<HTMLButtonElement>(
+        'disabled-button-overlay',
+        disabledMsg !== undefined,
+        false
+    );
+    const popover = useMemo(
+        () => (
+            <Popover id="disabled-button-popover" title={title} placement="bottom" targetRef={targetRef}>
+                {disabledMsg}
+            </Popover>
+        ),
+        [disabledMsg, title]
+    );
 
+    // Note: we use onPointerEnter/Leave so events propagate when the button is disabled
     return (
-        <>
-            {disabledMsg ? (
-                <OverlayTrigger
-                    placement="bottom"
-                    overlay={
-                        <Popover id="disabled-button-popover" title={title}>
-                            {disabledMsg}
-                        </Popover>
-                    }
-                >
-                    <div className={'disabled-button-with-tooltip ' + className}>
-                        <button className={`btn btn-${bsStyle}`} disabled type="button">
-                            {children}
-                        </button>
-                    </div>
-                </OverlayTrigger>
-            ) : (
-                <button className={`${className} btn btn-${bsStyle}`} onClick={onClick} type="button">
-                    {children}
-                </button>
-            )}
-        </>
+        <button
+            className={`${className} btn btn-${bsStyle}`}
+            disabled={disabledMsg !== undefined}
+            onClick={onClick}
+            onPointerEnter={onMouseEnter}
```

Comment:
Interesting. Should we just stop using `onMouseEnter` and `onMouseLeave` all together in favor of the pointer variants?

Reply:
Possibly, but I want to do some more research to see if there are any caveats to using these events